src/tutorials/code/node/rest/5.js

```diff
@@ -1,19 +1,19 @@
 'use strict';
 
-const Beam = require('beam-client-node');
+const Mixer = require('beam-client-node');
 
-const beam = new Beam();
+const client = new Mixer.Client();
```

Comment:
I think this should be `const client = new Mixer.Client(new Mixer.DefaultRequestRunner());`